cli/src/api-key-success.html

```diff
@@ -1,254 +1,173 @@
-            fill="#2D3748"
-          />
-          <path
-            d="M160.9 233.889L0.461715 234C0.339494 234 0.222403 233.954 0.136105 233.874C0.0498068 233.793 0.00134277 233.684 0.00134277 233.57C0.00134277 233.457 0.0498068 233.348 0.136105 233.267C0.222403 233.187 0.339494 233.141 0.461715 233.141L160.9 233.03C161.022 233.03 161.139 233.076 161.226 233.156C161.312 233.237 161.361 233.346 161.361 233.46C161.361 233.573 161.312 233.682 161.226 233.763C161.139 233.843 161.022 233.889 160.9 233.889V233.889Z"
-            fill="#CACACA"
-          />
-          <path
-            d="M47.5654 79.7206C47.9214 79.1997 48.0985 78.6061 48.0578 78.0704C48.0171 77.5347 47.7619 77.1008 47.3483 76.8641L46.0058 78.8282L47.5654 79.7206Z"
-            fill="#06D6A0"
-          />
-          <path
-            d="M43.0221 80.7089C42.4559 80.3866 42.0161 79.9259 41.7993 79.4282C41.5824 78.9304 41.6064 78.4363 41.866 78.0546L44.0006 79.2698L43.0221 80.7089Z"
-            fill="#FFBB00"
-          />
-          <path
-            d="M40.8843 76.4171C41.0338 77.0139 41.4358 77.5302 42.002 77.8525L42.002 77.8524L44.1364 79.0674C44.4911 78.5457 44.6064 77.9174 44.4569 77.3206C44.3074 76.7239 43.9053 76.2076 43.3392 75.8853L43.3392 75.8853L41.2048 74.6703C40.85 75.192 40.7348 75.8204 40.8843 76.4171Z"
-            fill="#FF0075"
-          />
-          <path
-            d="M47.4554 74.992C47.6034 75.5891 47.4865 76.2172 47.1304 76.7382L47.1304 76.7381L45.7881 78.702C45.2228 78.3785 44.8221 77.8613 44.6741 77.2642C44.5261 76.6671 44.643 76.039 44.999 75.5181L44.9991 75.5182L46.3413 73.5543C46.9066 73.8778 47.3074 74.395 47.4554 74.992Z"
-            fill="#0076FF"
-          />
-        </g>
+
+<head>
+  <title>API Key generated successfully</title>
+  <style>
+    @font-face {
+      font-family: Inter;
+      font-weight: 100 900;
+      font-display: block;
+      font-style: normal;
+      font-named-instance: 'Regular';
+      src: url('https://app.xata.io/assets/fonts/Inter.woff2?v=3.15') format('woff2');
+      font-display: swap;
+    }
+
+    body {
+      font-family:
+        Inter,
+        -apple-system,
+        BlinkMacSystemFont,
+        'Segoe UI',
+        Roboto,
+        Helvetica,
+        Arial,
+        sans-serif,
+        'Apple Color Emoji',
+        'Segoe UI Emoji',
+        'Segoe UI Symbol';
+      height: 100vh;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-around;
+      background: #0f0c1b;
+      color: #f0edff;
+      padding: 1rem;
+    }
+
+    .box {
+      width: 600px;
+      max-width: 100%;
+      margin: auto;
+    }
+
+    .logo {
+      display: flex;
+      justify-content: center;
+      margin-bottom: 3rem;
+    }
+
+    pre,
+    code {
+      margin: 0;
+      padding: 0;
+    }
+
+    pre {
+      background: rgba(255, 255, 255, 0.05);
+      padding: 1rem;
+      overflow-x: auto;
+      border-radius: 4px;
```

Comment:
@snide Has this been intentionally left in `px`? Just checking as I see you're moving most of the other dimensions to `rem`?

Reply:
No particular reason. This page is  it's a hard coded value either way. I'll shift if to 0.25rem which will be the same value.